aws/sentry/aurora-postgres.tf

```diff
@@ -0,0 +1,204 @@
+variable "postgres_admin_password" {
+  type        = string
+  description = "Postgres password for the admin user"
+  default     = ""
+}
+
+variable "postgres_db_name" {
+  type        = string
+  description = "Postgres database name"
+  default     = "sentry"
+}
+
+variable "aurora_postgres_engine_version" {
+  type        = string
+  description = "Database Engine Version for Aurora PostgeSQL"
+  default     = "9.6.12"
+}
+
+variable "aurora_postgres_cluster_family" {
+  type        = string
+  description = "Database Engine Version for Aurora PostgeSQL"
+  default     = "9.6.12"
+}
+
+# db.r4.large is the smallest instance type supported by Aurora Postgres 9.6
+# Postgres 10.6 and later can use db.r5.large
+# Postgres 10.7 and later can use db.t3.medium
+# https://aws.amazon.com/rds/aurora/pricing
+variable "postgres_instance_type" {
+  type        = string
+  default     = "db.r4.large"
+  description = "EC2 instance type for Postgres cluster"
+}
+
+variable "postgres_cluster_size" {
+  type        = number
+  default     = "2"
+  description = "Postgres cluster size"
+}
+variable "postgres_autoscaling_enabled" {
+  type        = bool
+  default     = false
+  description = "Set true to enable the database cluster autoscaler"
+}
+variable "postgres_autoscaling_min_capacity" {
+  type        = number
+  default     = 1
+  description = "Minimum number of Postgres instances to be maintained by the autoscaler"
+}
+variable "postgres_autoscaling_max_capacity" {
+  type        = number
+  default     = 6
+  description = "Maximum number of Postgres instances to be maintained by the autoscaler"
+}
+
+variable "postgres_cluster_enabled" {
+  type        = bool
+  default     = "true"
+  description = "Set to false to prevent the module from creating any resources"
+}
+
+variable "postgres_iam_database_authentication_enabled" {
+  type        = string
+  default     = "false"
+  description = "Specifies whether or mappings of AWS Identity and Access Management (IAM) accounts to database accounts is enabled."
+}
+
+resource "random_string" "postgres_admin_password" {
+  count   = local.postgres_cluster_enabled ? 1 : 0
+  length  = 26
+  special = true
+}
+
+locals {
+  # Some of these locals used to be computed. Keep them as local just for consistency,
+  # even though now they are just copies of variables.
+  postgres_cluster_enabled = var.postgres_cluster_enabled
+  postgres_db_name         = var.postgres_db_name
+  postgres_admin_user      = length(var.postgres_admin_user) > 0 ? var.postgres_admin_user : "sentry"
+  postgres_admin_password  = length(var.postgres_admin_password) > 0 ? var.postgres_admin_password : join("", random_string.postgres_admin_password.*.result)
+}
+
+module "aurora_postgres" {
+  source           = "git::https://github.com/cloudposse/terraform-aws-rds-cluster.git?ref=tags/0.16.0"
+  namespace        = var.namespace
+  stage            = var.stage
+  name             = var.postgres_name
```

Comment:
since this module is for sentry, maybe `var.postgres_name` should be `sentry` and remove the attributes.

Reply:
No, because then the postgres name would be "cathex-corp-sentry", which would be confusing.

Reply:
not sure why it would be confusing since it's a postgres database (I mean it's not necessary to add `postgres` to the name since it's obvious from the context)